kuma/search/tests/test_serializers.py

```diff
@@ -59,6 +59,25 @@ def test_document_serializer(self):
         assert isinstance(dict_data, dict)
         assert dict_data['id'] == result[0].id
 
+    def test_search_query_serializer(self):
+        search_serializer = SearchQuerySerializer(
+            data={'q': 'test'}
+        )
+        self.assertTrue(search_serializer.is_valid())
+        assert {} == search_serializer.errors
```

Comment:
This notation comes from PHP whereby writing it this way protects you from accidentally making an assignment. E.g. 
```php
if ([] == someThing) {
   ...
}
```
In Python, we always do it the other way around. That puts the mental focus first on the the left-hand side which is what's being tested.